### DISCUSSION TITLE
Database connection error after changing root folder name

### DISCUSSION THREAD

**Discussion by dgrcode** [prisma/prisma#14549] [Q&A] [answered]
I was having issues after changing the root folder name of my project. Suddenly db connections were failing. The error I was getting was:
```
Message: Error in Prisma Client request: 


Invalid `prisma.user.findMany()` invocation:


  Database `mydb` does not exist on the database server at `localhost:5432`.
```

This might be related to docker somehow. If that's the case, I should update the title to make this easier to be found.

I couldn't find any mention about this scenario in the docs/issues/discussions, so I wanted to post how I've solved it to help anyone looking for this.

How I solved it was:
```
npx prisma generate
npx prisma db push
```

I don't really know if just running `npx prisma db push` would have fixed it, but I run `generate` first because I figured the connection urls were outdated in the client. But just `generate` didn't fix the problem, and after `db push` it worked again.

I hope this helps someone!

**Comment by nurul3101** [comment, marked as answer]
Hey @dgrcode 👋 
Thanks for sharing the solution here, this would be helpful to someone who might run into this issue! 🙌

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
